### DISCUSSION TITLE
Relay examples

### DISCUSSION THREAD

**Discussion by ivosabev** [TanStack/router#69] [Ideas]
Can we get Relay examples, seems the router could support it with async loading and without waterfalls. I believe everyone using Relay would be very happy if there is finally a router that will have a good support.

**Comment by tannerlinsley** [comment]
I think this would be awesome, but I don't have any Relay experience.

**Comment by axhl** [comment]
As an intermediary step to @ivosabev's sound suggestion I'm cross-referencing this with some open Relay issues to draw more community eyeballs to React Location, as it looks like the library will fill a real gap: https://github.com/facebook/relay/issues/3449 https://github.com/facebook/relay/issues/3547.
